requirements.txt

```diff
@@ -4,3 +4,4 @@ prettyprinter
 ruamel.yaml
 ruamel.yaml.clib>=0.2
 pydocstyle==5.0.1
+antlr4-python3-runtime
```

Comment:
What is this for?

Reply:
The relay text parser.  The VM loads some ressources from a text file that needs to be parsed.

Reply:
Shouldn't that dependency be declared by the tvm/relay package?